convert-hf-to-gguf.py

```diff
@@ -30,6 +31,42 @@
 logger = logging.getLogger("hf-to-gguf")
 
 
+@dataclass
+class Metadata:
+    name: Optional[str] = None
+    author: Optional[str] = None
+    version: Optional[str] = None
+    url: Optional[str] = None
+    description: Optional[str] = None
+    licence: Optional[str] = None
```

Comment:
This typo also exists in `convert.py`, I think.

```suggestion
    license: Optional[str] = None
```